src/diffusers/pipelines/wuerstchen/common.py

```diff
@@ -0,0 +1,95 @@
+import torch
+import torch.nn as nn
+
+from ...models.attention_processor import Attention
+
+
+class LayerNorm2d(nn.LayerNorm):
```

Comment:
Such a function should not exist (think it's the 3rd time I write this) - let's write it out whenever we use layernorm